src/validation.h

```diff
@@ -557,7 +557,14 @@ static const unsigned int REJECT_CONFLICT = 0x102;
 /** Dump the mempool to disk. */
 void DumpMempool();
 
+/** Result states for loading the mempool from disk */
+enum LoadMempoolResult {
```

Comment:
If user aborts at T=5, and doesn't shut down the node, when it *does* finally shutdown at T=5000, you would want to dump it. This isn't possible, but the enum name doesn't make that clear. So a simply two-value enum with DoNotSave vs PleaseSave might be best?